### DISCUSSION TITLE
Impact Based Warning property?

### DISCUSSION THREAD

**Discussion by CASAelyons** [weather-gov/api#651] [Ideas]
Any plans to include IBW info in certain alerts?  My apologies if this is included somehow or has been superceded by something like the severity.

**Comment by sullynole** [comment]
IBW info can be found under the parameters field.  See here for CAP documentation:  https://vlab.noaa.gov/web/nws-common-alerting-protocol/cap-documentation

**Reply by CASAelyons** [reply]
Perfect.  Thank you for pointing that out.

**Reply by CASAelyons** [reply]
Once again thanks.  I see now that the YAML has the following:
parameters:
          type: object
          description: "System-specific additional parameters associated with the alert message.\nThe keys in this object correspond to parameter definitions in the NWS CAP specification.\n"
          additionalProperties:
            type: array
            items: {  }

I'd suggest perhaps a link to the CAP specification in here, and possibly a high level sentence or two about what additional info it contains.  All the same, I think this definitely does have what I was looking for and I appreciate the timely response.
